lib/pengine/bundle.c

```diff
@@ -1491,10 +1491,19 @@ pe__bundle_text(pcmk__output_t *out, va_list args)
     GList *only_node = va_arg(args, GList *);
     GList *only_rsc = va_arg(args, GList *);
 
+    const char *desc = NULL;
+
     pe__bundle_variant_data_t *bundle_data = NULL;
     int rc = pcmk_rc_no_output;
     gboolean print_everything = TRUE;
 
+    // User-supplied description
+    if (pcmk_is_set(show_opts, pcmk_show_rsc_only)
```

Comment:
With the new flag, this will become `pcmk_any_flags_set(show_options, pcmk_show_rsc_only|pcmk_show_description)` (here and the other similar checks for description)